README.md

```diff
@@ -0,0 +1,21 @@
+# Villainary
+Villainary is a technical project built with Next.js, aiming to showcase an array of intriguing villains.  The project may grow to include various backend and frontend technologies to demonstrate mastery across multiple paradigms.
```

Comment:
This is not accurate if being added at the root level. Only villainary react is developed using next JS. I see the second line includes information about multiple paradigms, but I think the overall description misses the point of villainary. Please relabel and rescope this to villainary-react